src/test/java/com/easypost/Fixture.java

```diff
@@ -343,4 +344,26 @@ public static Map<String, Object> basicOrder() {
 
         return order;
     }
+
+    /**
+     * Basic end shipper address that is reusable in all test coverage.
+     *
+     * @return A map that has the end shipper address info filled.
+     */
+    public static Map<String, Object> basicEndShipperAddress() {
```

Comment:
Just call it `endShipperAddress` because all fields are always required meaning there won't ever be multiple kinds based on the filled in data.

```suggestion
    public static Map<String, Object> endShipperAddress() {
```